ccm_web/package-lock.json

```diff
@@ -1,8 +1,7137 @@
 {
   "name": "ccm_web",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@chrisrrowland was this file auto generated when you doing a docker build or you had to run the `npm install` manually on your machine?

Reply:
I don't think a `package-lock.json` change in the container would be propagated to the repo through any volume mapping, so must have been done locally?

Reply:
@ssciolla I think the way the Docker file is setup is it is taking both package.json and package-lock.json and when build happens it will always go by the lock file dependency tree. We might miss the getting any latest updates on package. 
How do the package update happens automatically?

Reply:
package-lock.json already existed.  I added material-ui dependencies which accounted for most of the changes.  I believe I also did an npm install on eslint when I didn't realize it was already installed, resulting in an upgrade there.
Typescript also got an upgrade, but I don't recall why.  I don't think it hurts though.

Reply:
Yeah.. the process is same with other project. 
But does it make sense to automate the process.
1. During the docker build don't add the lock file during the npm install. This will get all the latest package updates
2. This creates a new lock file and once the build is done due to volume mapping the new lock file is updated or mapped local.
3. This way we will always end up will latest dependencies
4. This is just a proposal, no work is needed at this moment.

Reply:
Yeah, I need to think about this more, and may want to discuss. I kind of like the current approach.

Reply:
The purpose of the package-lock file is so you don't get unintended upgrades, right?
I don't see what is not working with the current setup.

Reply:
I agree. Anyways, let's discuss separately, nothing you need to do here.